pkg/grpc/utils.go

```diff
@@ -37,33 +36,35 @@ var RPCTimeout = DefaultRPCTimeout
 
 var log = ctrl.Log.WithName("util")
 
-// CreateGrpcConnection create a grpc connection with given port
-func CreateGrpcConnection(ctx context.Context, c client.Client, pod *v1.Pod, port int) (*grpc.ClientConn, error) {
-	nodeName := pod.Spec.NodeName
-	log.Info("Creating client to chaos-daemon", "node", nodeName)
-
-	ns := config.ControllerCfg.Namespace
-	var endpoints v1.Endpoints
-	err := c.Get(ctx, types.NamespacedName{
-		Namespace: ns,
-		Name:      "chaos-daemon",
-	}, &endpoints)
-	if err != nil {
-		return nil, err
-	}
+// CreateGrpcConnectionWithAddress create a grpc connection with given port and address
+func CreateGrpcConnectionWithAddress(address string, port int, caCert string, cert string, key string) (*grpc.ClientConn, error) {
```

Comment:
`caCert`, `cert`, and `key` are paths of these files, please present this meaning in argument name, like `caCertFile`.

Reply:
Fixed